trunk/src/LV2/gx_amp.lv2/gx_amp.ttl

```diff
@@ -319,6 +319,26 @@ https://sourceforge.net/apps/mediawiki/guitarix/index.php?title=Cabinet_Impulse_
         lv2:minimum 0.0 ;
         lv2:maximum 1.0 ;
         lv2:portProperty lv2:toggled;
+    ] ,	[
+        a lv2:ControlPort, 
+            lv2:InputPort ;
+        lv2:index 19 ;
+        lv2:symbol "placeholder_out" ;
+        lv2:name "Placeholder Out" ;
+    ] , [
+        a lv2:ControlPort, 
+            lv2:InputPort ;
+        lv2:index 20 ;
+        lv2:symbol "placeholder_in" ;
+        lv2:name "Placeholder in" ;
```

Comment:
We shout remove AMP_OUTPUT1 and AMP_INPUT1 from gxamp.h, as they are unused artefacts, instead using placeholders here.